cmd/torpedo/torpedo.go

```diff
@@ -26,13 +27,15 @@ const (
 	nodeDriverCliFlag    = "node-driver,n"
 	storageDriverCliFlag = "storage,v"
 	testsCliFlag         = "tests,t"
+	logLocationCliFlag   = "logLocation"
```

Comment:
we should have the param name as log-location as this is a cli param and to make it consistent with other params

Reply:
Will do.